src/components/Tool/FixPhoneNumbers/Contact.tsx

```diff
@@ -234,13 +255,15 @@ const Contact: React.FC<Props> = ({
                                 className={classes.hoverHighlight}
                               />
                             ) : (
-                              <StarOutlineIcon
-                                data-testid={`starOutlineIcon-${personId}-${index}`}
-                                className={classes.hoverHighlight}
-                                onClick={() =>
-                                  handleChangePrimary(personId, index)
-                                }
-                              />
+                              <Tooltip title="Set as Primary">
+                                <StarOutlineIcon
```

Comment:
Hey man! This is looking great already. I noticed a few things are off styling-wise with the locked phone numbers.

- Could you center the stars under the word Primary?
- Could you make the lock aligned better like you have the trash can icon
- Could you make sure the new phone number row stays as long as the rows above it? Seems like this only happens when locked rows are above it?

<img width="716" alt="Screenshot 2024-08-08 at 7 01 03 PM" src="https://github.com/user-attachments/assets/ddefe26e-4ec4-4f8e-8ca8-24f934b52fbf">

Reply:
<img width="733" alt="Screenshot 2024-08-08 at 7 05 13 PM" src="https://github.com/user-attachments/assets/a8fc71dd-9133-43e5-89b9-75677a62883e">